### ISSUE TITLE
WhitespaceAroundCheck: fix for anonymous inner class

### ISSUE THREAD

**Issue by daniilyar** [checkstyle/checkstyle#105]
Anonymous inner class syntax is not well recognized by WhitespaceAroundCheck, which causes false-positives:

Example:

``` java
new Thread() {
public void run() { 
// ... 
}
}.start();
Example 2 (very common in Gui code): 
new SwingWorker() {
public void construct() { 
// ... 
}
}.start();
```

==> wrong error "missing a space after '}'

The following correction should work and fix it:

in WhitespaceAroundCheck, line 256, instead of:

``` java
&& !((type == TokenTypes.RCURLY)
&& ((nextChar == ')')
|| (nextChar == ';')
|| (nextChar == ','))))
```

use the following:

``` java
&& !((type == TokenTypes.RCURLY)
&& ((nextChar == ')')
|| (nextChar == ';')
|| (nextChar == ',')
|| (nextChar == '.'))))
```
